locales/fr_FR/pcsx2_Main.po

```diff
@@ -28,13 +28,11 @@ msgstr "Erreur critique."
 
 #: common/include/Utilities/Exceptions.h:255
 msgid "Parse error"
-msgstr "Parse error"
+msgstr "erreur d'analyse"
 
 #: common/include/Utilities/Exceptions.h:279
 msgid "Your machine's hardware is incapable of running PCSX2.  Sorry dood."
-msgstr ""
-"Le matériel de votre ordinateur rend impossible le fonctionnement de PCSX2. "
-"Bien essayé !"
+msgstr "Le matériel de votre ordinateur rend impossible le fonctionnement de PCSX2. Bien essayé !"
```

Comment:
```
Bien essayé  => Desole 
```

Reply:
Oui, ceci semble plus approprié, mais n'oubliez pas les accents :), desole doit etre changé en: désolé.
"Yes, it is more correct but don't forget some french rules :), desole must be change to: désolé."